pkg/operator2/deployment.go

```diff
@@ -8,15 +8,14 @@ import (
 	"strings"
 
 	operatorv1 "github.com/openshift/api/operator/v1"
-	authv1alpha1 "github.com/openshift/cluster-authentication-operator/pkg/apis/authentication/v1alpha1"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
-const hashAnnotation = authv1alpha1.GroupName + "/rvs-hash"
+var hashAnnotation = operatorv1.GroupName + "/rvs-hash"
```

Comment:
@sallyom as a followup, make `operatorv1.GroupName` a `const` in `openshift/api` and then undo this change.